plugins/http_plugin/tests/unit_tests.cpp

```diff
@@ -624,6 +623,12 @@ BOOST_FIXTURE_TEST_CASE(bytes_in_flight, http_plugin_test_fixture) {
       return count_of_status_replies;
    };
 
+   auto wait_for_no_bytes_in_flight = [&](uint16_t max = std::numeric_limits<uint16_t>::max()) {
```

Comment:
`uint16_t max = std::numeric_limits<uint16_t>::max()` is kinda weird.  Why not use `size_t` and an constant like `10000`?

Reply:
Just following the pattern of `drain_http_replies`.